airflow-core/src/airflow/ui/src/pages/DagsList/DagsFilters/StateFilters.tsx

```diff
@@ -65,6 +74,17 @@ export const StateFilters = ({ isAll, isFailed, isRunning, isSuccess, onStateCha
         <StateBadge state="success" />
         {translate("common:states.success")}
       </QuickFilterButton>
+      <QuickFilterButton
+        data-testid="dags-needs-review-filter"
+        isActive={needsReview}
+        onClick={onStateChange}
+        value="needs_review"
+      >
+        <StateBadge colorPalette="deferred">
+          <LuUserRoundPen />
+        </StateBadge>
+        {translate("hitl:needsReview")}
```

Comment:
Interesting, code looks good but on my instance the I18n label is not found, even after I was hitting F5 multiple times...

<img width="553" height="186" alt="Image" src="https://github.com/user-attachments/assets/e04af0cc-a71e-4f0f-8665-c057d45888b0" />